File: easy/cielab/cielab.c
/*
    started at: Saturday, 14. December 2013 04:08PM
*/

#include <stdio.h>

int main(int argc, char const *argv[])
{
    int num1 = 0;
    int num2 = 0;
    int num;
    scanf("%d",&num1);
    scanf("%d",&num2);
    num = num1-num2;
    printf("%d\n",num%10==9 ? num-1 : num+1);
    return 0;
}

File: easy/cielab/cielab.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Saturday, 14. December 2013 04:08PM
import sys

def main():
    nums = map(int,sys.stdin.readline().split())
    num = nums[0]-nums[1]
    print num+(-1 if num%10==9 else 1)

if __name__ == '__main__':
    main()